website/src/app/kb/user-guides/linux-client/readme.mdx

```diff
@@ -1,7 +1,7 @@
 # Linux Client
 
-Firezone supports Linux with a native client for ARM v7, ARM64, and Intel x64
-architectures.
+Firezone supports Linux with a native, headless client for ARM v7, ARM64, and
+Intel x64 architectures.
```

Comment:
Like test on 32-bit x86, or test on AMD CPUs? The latter might be more feasible when we have the baremetal testbed running well (Ryzen)

Reply:
AMD CPUs. I'm okay with giving up on 32-bit x86 I see Vultr has some compute instances that are AMD too, and they have cloud-init so that could be used to automatically build and run smoke tests or something.

Reply:
Hm, out of curiosity, what would be the reason to test on different vendors of the same architecture? To uncover µarch bugs?

Reply:
We could say "Intel/AMD x86_64" without testing, I just don't like calling it Intel 64 when AMD actually invented it and then Intel followed second.

Reply:
I see what you mean. Updated to `x86-64`